desktop/ui/src/main/java/org/datacleaner/connection/DatastoreDescriptorDesktopBindings.java

```diff
@@ -0,0 +1,186 @@
+
+    private static final DatastoreDescriptor SUGARCRM_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl("SugarCRM",
+            "Connect to a SugarCRM system", SugarCrmDatastore.class);
+
+    private static final DatastoreDescriptor MONGODB_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(
+            "MongoDB database", "Connect to a MongoDB database", MongoDbDatastore.class);
+
+    private static final DatastoreDescriptor COUCHDB_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(
+            "CouchDB database", "Connect to an Apache CouchDB database", CouchDbDatastore.class);
+
+    private static final DatastoreDescriptor ELASTICSEARCH_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(
+            "ElasticSearch index", "Connect to an ElasticSearch index", ElasticSearchDatastore.class);
+
+    private static final DatastoreDescriptor CASSANDRA_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(
+            "Cassandra database", "Connect to an Apache Cassandra database", CassandraDatastore.class);
+
+    private static final DatastoreDescriptor HBASE_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl("HBase database",
+            "Connect to an Apache HBase database", HBaseDatastore.class);
+
+    private static final DatastoreDescriptor HIVE_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(
+            DatabaseDriverCatalog.DATABASE_NAME_HIVE, "Connect to an Apache Hive database", JdbcDatastore.class);
+
+    private static final DatastoreDescriptor MYSQL_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(DatabaseDriverCatalog.DATABASE_NAME_MYSQL,
+            "Connect to a MySQL database", JdbcDatastore.class);
+
+    private static final DatastoreDescriptor POSTGRESQL_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(DatabaseDriverCatalog.DATABASE_NAME_POSTGRESQL,
+            "Connect to a PostgreSQL database", JdbcDatastore.class);
+
+    private static final DatastoreDescriptor ORACLE_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(DatabaseDriverCatalog.DATABASE_NAME_ORACLE,
+            "Connect to a Oracle database", JdbcDatastore.class);
+
+    private static final DatastoreDescriptor SQLSERVER_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl(DatabaseDriverCatalog.DATABASE_NAME_MICROSOFT_SQL_SERVER_JTDS,
+            "Connect to a Microsoft SQL Server database", JdbcDatastore.class);
+    
+    private static final DatastoreDescriptor COMPOSITE_DATASTORE_DESCRIPTOR = new DatastoreDescriptorImpl("Composite datastore",
+            "Create a composite datastore", CompositeDatastore.class);
+
+    private static Map<DatastoreDescriptor, String> _iconPaths = new HashMap<DatastoreDescriptor, String>();
+
+    private static Map<DatastoreDescriptor, Class<? extends AbstractDatastoreDialog<? extends Datastore>>> _dialogClasses = new HashMap<>();
+
+    static {
+        _iconPaths.put(CSV_DATASTORE_DESCRIPTOR, IconUtils.CSV_IMAGEPATH);
```

Comment:
If you make DatastoreDescriptor just a desktop-ui-thing, then you can add the icon paths directly on the DatastoreDescriptor class.